doc/source/general/cpu_hardware.rst

```diff
@@ -78,12 +73,6 @@ other way around.
 Electric Drain
 --------------
 
-.. versionadded:: 0.19.0
-    As of version 0.19.0, the electric charge drain varies depending
-    on CPU % usage.  Prior to version 0.19.0, the CPU load made no
-    difference and the electric drain was constant regardless of
-    utilization.
-
```

Comment:
Another reword, but earlier in the section as part of the second paragraph which mentions micropayments, since it shows how the evolution of the feature has changed.